server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -362,8 +367,8 @@ export class RestGitService {
         }).catch(getRequestErrorTranslator(url, "POST"));
     }
 
-    private async delete<T>(url: string): Promise<T> {
-        return this.restWrapper.delete<T>(url)
+    private async delete<T>(url: string, headers?: any): Promise<T> {
+        return this.restWrapper.delete<T>(url, undefined, headers)
```

Comment:
Just to confirm, `RestWrapper` already provides support for headers, right?

Reply:
correct, RestWrapper/BasicWrapper respects passed in headers.